### ISSUE TITLE
system_cmd.c waf compile error on pi4

### ISSUE THREAD

**Issue by okyeron** [monome/norns#925]
https://github.com/monome/norns/blob/e22003b050d27149e4150bc4c53d7b2a0afb7b47/matron/src/system_cmd.c#L91

```
../matron/src/system_cmd.c: In function ‘run_cmd’:
../matron/src/system_cmd.c:91:5: error: ‘strncpy’ output truncated before terminating nul copying as many bytes from a string as its length [-Werror=stringop-truncation]
     strncpy(cap, capture, len);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~
../matron/src/system_cmd.c:89:18: note: length computed here
     size_t len = strlen(capture);
                  ^~~~~~~~~~~~~~~
cc1: all warnings being treated as errors

Waf: Leaving directory `/home/we/norns/build'
Build failed
 -> task in 'matron' failed with exit status 1 (run with -v to display more information)
```

**Comment by okyeron** [comment]
pi4 has gcc 8.3.0 

found a [stackoverflow reference](https://stackoverflow.com/questions/54902205/gcc-8-wstringop-truncation-inconsistent-behavior) to similar.

another [reference here](https://github.com/WizardMac/ReadStat/issues/151#issuecomment-506952159). Not entirely sure that applies?

**Comment by catfact** [comment]
most oddly, i can't seem reproduce with gcc 8.3.0 on ubuntu. (with `-Wall -Werror -O3` etc)

anyways, i guess PR #926 will do it

**Comment by okyeron** [comment]
fixed with #926
